test/test_multidevice_pipeline.cpp

```diff
@@ -304,6 +304,122 @@ INSTANTIATE_TEST_SUITE_P(
         ::testing::Values(true),
         ::testing::Bool()));
 
+enum class SchedulingMode {
+  noIntraDeviceScheduling,
+  manualScheduling,
+};
+
+std::ostream& operator<<(std::ostream& out, const SchedulingMode& mode) {
+  switch (mode) {
+    case SchedulingMode::noIntraDeviceScheduling:
+      return out << "noIntraDeviceScheduling";
+    case SchedulingMode::manualScheduling:
+      return out << "manualScheduling";
+    default:
+      NVF_ERROR(false);
+  }
+  return out;
+}
+
+using PipelineTestStagedReductionParams = std::tuple<SchedulingMode>;
+class PipelineTestStagedReduction
+    : public PipelineTest,
+      public ::testing::WithParamInterface<PipelineTestStagedReductionParams> {
+};
```

Comment:
```suggestion
class PipelineTestStagedReduction
    : public PipelineTest,
      public ::testing::WithParamInterface<SchedulingMode> {
};
```

Is there a good reason for a single-element tuple (and therefore indirection)?